cg/services/pacbio/metrics/models.py

```diff
@@ -63,24 +63,22 @@ class ProductivityMetrics(BaseModel):
     p_0: int = Field(..., alias=LoadingAttributesIDs.P_0)
     p_1: int = Field(..., alias=LoadingAttributesIDs.P_1)
     p_2: int = Field(..., alias=LoadingAttributesIDs.P_2)
+    percentage_p_0: float
```

Comment:
Can these be given more descriptive names?

Reply:
These are the names of the PacBio attributes that we want to Parse. We agreed not to include unnecessary biological context at this level as we are only parsing data
<img width="241" alt="Screenshot 2024-07-18 at 16 29 30" src="https://github.com/user-attachments/assets/31b63029-84ed-4005-9253-261421661a6d">

Reply:
I think the alias decouples us from having to name the field the same way. I think having the domain language in the 
models is a good thing, better than `p_0` etc. Maybe you can discuss it with fellow project members :)